### PR TITLE
Logic added to the Calculator App

### PR DESCRIPTION
**Pull request by javierbotero**
This pull request creates the logic of the calculator app implementing two JavaScript modules: calculate.js and operate.js which will be responsible for all calculations and those are included in the App component.

### CODE REVIEW

**Comment by sumancrest0001** [comment]
### Status: changes required :gear:

**Nice work @javierbotero :clap: ,**
We have to make a small change to get approved.
[Required] It seems that you forgot to import calculate.js in the App component. [Reference](https://microverse.pathwright.com/library/react-redux-v11/184438/path/step/106376018/)

**Comment by sumancrest0001** [comment]
[Optional] Please commit your code as frequently as possible which helps you to go back easily.

**Review by sumancrest0001 — CHANGES_REQUESTED** [review]
**Please make the necessary changes and submit the project for another review.
If you have any queries regarding the review, do message me on Slack.**

**Happy coding !!! :slightly_smiling_face:**

File: src/logic/calculate.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import operate from './operate';
+
+export default function calculate(data, ButtonName) {
```
Comment (sumancrest0001) [inline src/logic/calculate.js]: [Required] Please follow the JavaScript naming convention. **ButtonName** should be camelCase.

**Review by Forison — APPROVED** [review]
## Status::Approved✔️
#### Good job on addressing the issue raised in the previous review, calculator has been imported in App.jsx, kindly go ahead and , merge the branch and proceed to the next milestone👏👏👏.
#### Happy coding👨‍💻
Add Forison

**Pull request merged**